helm/hpcc/templates/_warnings.tpl

```diff
@@ -52,4 +52,79 @@ Pass in dict with root and warnings
   {{- $_ := set $warning "msg" (printf "The configuration contains ephemeral planes: %s" (print (uniq $match.ephemeral)) )  -}}
   {{- $_ := set $ctx "warnings" (append $ctx.warnings $warning) -}}
  {{- end -}}
+ {{- /* Warn if resources not defined for any cost component or default cpuRate being used */ -}}
+ {{- if eq .root.Values.global.cost.perCpu 0.0565000000001 -}}
+  {{- $_ := set $ctx "usingDefaultCpuCost" "true" -}}
+ {{- end -}}
+ {{- $_ := set $ctx "missingResources" list -}}
+ {{- $_ := set $ctx "defaultCpuRateComponents" list -}}
+ {{- range $component := .root.Values.eclagent -}}
+  {{- if and (not $component.disabled) (eq $component.type "hthor" ) -}}
+   {{- $hasResources := include "hpcc.checkResources" (dict "resources" $component.resources) -}}
```

Comment:
Could pass $component instead of dict "resources" $component.resources.  But this is consistent with the more complex cases, so I think it make sense.

Reply:
Yes, I did it like this because there are components like Thor that have multiple resource objects.